### ISSUE TITLE
Increase timeout for localhost network

### ISSUE THREAD

**Issue by fvictorio** [nomiclabs/hardhat#1549]
A lot of users are using the `localhost` network with a forked hardhat node. This frequently leads to timeouts for the first run. We should consider just increasing it to a larger value.

**Comment by alcuadrado** [comment]
We should try to reproduce it, so that we have a better idea about which timeout to use. Now it's 20sec.

**Comment by fvictorio** [comment]
We are going to increase the localhost and mocha timeouts to 40s.

**Comment by zendevil** [comment]
Is there a way to manually increase the timeout? I'm getting timeouts running my tests.

**Comment by fvictorio** [comment]
> Is there a way to manually increase the timeout? I'm getting timeouts running my tests.

```js
networks: {
  localhost: {
    url: "http://localhost:8545",
    timeout: 40000
  }
},
mocha: {
  timeout: 40000
}
```

**Comment by feuGeneA** [comment]
Released in [hardhat v2.8.3](https://github.com/nomiclabs/hardhat/releases/tag/hardhat%402.8.3)
